examples/example-gateway/endpoints/baz/baz_simpleservice_method_call_test.go

```diff
@@ -39,10 +39,11 @@ import (
 var testCallCounter int
 
 func call(
-	ctx context.Context, reqHeaders map[string]string, r *baz.BazRequest,
+	ctx context.Context, reqHeaders map[string]string, args *baz.SimpleService_Call_Args,
```

Comment:
It's auto generated by thriftrw, I guess the reason is to be able to tell the service, method from the name